nestkernel/model_manager.h

```diff
@@ -318,60 +246,39 @@ class ModelManager : public ManagerInterface
   static bool compare_model_by_id_( const int a, const int b );
 
   /**
-   * The list of clean node models. The first component of the pair is a
-   * pointer to the actual Model, the second is a flag indicating if
-   * the model is private. Private models are not entered into the
-   * modeldict.
+   * List of clean built-in node models.
    */
-  std::vector< std::pair< Model*, bool > > pristine_models_;
-
-  std::vector< Model* > models_; //!< List of available models
+  std::vector< Model* > builtin_node_models_;
```

Comment:
I am not certain that I like `builtin_...`. First, in Python etc builtin functions are used just as other functions, they just come from a specific place. But in NEST, the `builtin_`s are never used for anything except to clone models from it. Second, from the perspective of `nestkernel`,  also `modelsmodule` is just another module, and any user modules will register to `builtin_node_models` just as `modelsmodule`. So there is nothing "built-in" about the models here. 

How about `unmodified_node_models_`or `reference_node_models_` or `raw_node_models_`? Maybe @abigailm has an idea?

Reply:
OK.